operator/pkg/helmreconciler/wait.go

```diff
@@ -96,6 +98,8 @@ func WaitForResources(objects object.K8sObjects, client kube.Client,
 }
 
 func waitForResources(objects object.K8sObjects, cs kubernetes.Interface, l *progress.ManifestLog) (bool, []string, map[string]string, error) {
+	var hpa *v2.HorizontalPodAutoscaler
```

Comment:
Right now we are still supporting v2beta2 https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases, so this may not always be the case.

Reply:
Thanks, It have been made modifications to the v2beta2 version